Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls.WinForms/Microsoft.Toolkit.Win32.UI.Controls.WinForms.csproj

```diff
@@ -2,33 +2,24 @@
 
   <PropertyGroup>
     <TargetFramework>net462</TargetFramework>
-    <RootNamespace>Microsoft.Toolkit.Win32.UI.Controls</RootNamespace>
-    <AssemblyName>Microsoft.Toolkit.Win32.UI.Controls</AssemblyName>
+    <RootNamespace>Microsoft.Toolkit.Win32.UI.Controls.WinForms</RootNamespace>
```

Comment:
@nmetulev This is a breaking change (and major version), yes?

Reply:
Will need to update docs for WebView on account that the packages change. We'll also need something to redirect users to the next version and inform them the existing package is no longer updated